### PR TITLE
add CordaloLinearState test

### PR DESCRIPTION
**Pull request by cedricwalter**
add missing unit test

### CODE REVIEW

File: corda-common/src/test/java/ch/cordalo/corda/common/states/CordaloLinearStateTest.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            super(externalId);
+        }
+
+        public UnderTest() {
+        }
+
+        @NotNull
+        @Override
+        protected Parties getParties() {
+            return null;
```
Comment (lolo8304) [inline corda-common/src/test/java/ch/cordalo/corda/common/states/CordaloLinearStateTest.java]: Never return null here. 
We should validate this in the get participants method.

File: corda-common/src/test/java/ch/cordalo/corda/common/states/CordaloLinearStateTest.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+        public UnderTest(String externalId) {
+            super(externalId);
+        }
+
+        public UnderTest() {
+        }
+
+        @NotNull
+        @Override
```
Comment (lolo8304) [inline corda-common/src/test/java/ch/cordalo/corda/common/states/CordaloLinearStateTest.java:38]: Test case missing for getParticiosnts() method. 

Add a party here and validate if part of participants

**Review by lolo8304 — CHANGES_REQUESTED** [review]

File: corda-common/src/test/java/ch/cordalo/corda/common/states/CordaloLinearStateTest.java (hunk 1 above)
Comment (cedricwalter) [reply corda-common/src/test/java/ch/cordalo/corda/common/states/CordaloLinearStateTest.java]: it is a delegate of the class under test (a stub), if i code something here, it make also no sense to test it as the implementation is dummy.
I change it and add a unit test but it test nothing...

File: corda-common/src/test/java/ch/cordalo/corda/common/states/CordaloLinearStateTest.java (hunk 2 above)
Comment (cedricwalter) [reply corda-common/src/test/java/ch/cordalo/corda/common/states/CordaloLinearStateTest.java:38]: done

**Pull request merged**